C_sharp_exercises_2/C_sharp_exercises_2/Exercises4.cs

```diff
@@ -0,0 +1,71 @@
+﻿using C_sharp_exercises_2;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+//Exercise 4:
+//Create a class Book which should have author, title and createdDate properties.
+//Create 10 instances of Book class.
+//1.Filter latest 5 created Books.
+//2. Group the books which authors are same person
+//3. Find Books which title starts with A letter
+namespace C_sharp_exercises_2
+{
+    class Book
+    {
+        public string author;
+        public string title;
+        public DateTime BookDta;
+        public Book(string name, string title, DateTime BookDta)
+        {
+            this.author = name;
+            this.title = title;
+            this.BookDta = BookDta;
+        }
+    }
+    internal class Exercises4
+    {
+        public void filteringtoNameAndTitle()
+        {
+            IList<Book> bookList = new List<Book>() { };
+            DateTime dt = DateTime.Now;
+
+            var Book1 = new Book("Max", "ANgular", dt);
+            var Book2 = new Book("Raffaello", "css", dt);
+            var Book3 = new Book("Michelangelo", "js", dt);
+            var Book4 = new Book("Doremi", "c++", dt);
+            var Book5 = new Book("Fasollya", "react", dt);
+            var Book6 = new Book("Max", "ANgular", dt);
+            var Book7 = new Book("Samuel", "css", dt);
+            var Book8 = new Book("Joe", "Alchimic", dt);
+            var Book9 = new Book("Harry", "js", dt);
+            var Book10 = new Book("Bob", "css", dt);
+
+            bookList.Add(Book1);
```

Comment:
you can add the instance here imminently
for example
`bookList.Add(new Book("Max", "ANgular", dt);`